src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 500;
+    this.name = 'Test';
+    Animal.alive.push(this.name);
+  }
+  static deleteIndex = Animal.alive.findIndex((animal) => animal.health !== 'dead');
```

Comment:
The static deleteIndex property is unnecessary and not used anywhere in the code. Also, the findIndex method is incorrectly used here since it should be used inside a method when an animal's health is less than or equal to 0.